pkg/operator/resources/resources.go

```diff
@@ -39,19 +39,22 @@ import (
 	kcore "k8s.io/api/core/v1"
 )
 
```

Comment:
nit: Add a comment here that says that this function returns an error if the resource does not exist